### PR TITLE
Create Example Modal and add it to Storybook

### PR DESCRIPTION
**Pull request by SebastianStupp**

### CODE REVIEW

File: client/src/components/Modal.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+export default function DefaultModal() {
+  const [showModal, setShowModal] = React.useState(false);
+
+  const handleOnClick = () => {
+    if (!showModal) {
+      setShowModal(true);
+    } else {
+      setShowModal(false);
+    }
+  };
```
Comment (phlgr) [inline client/src/components/Modal.js:56]: You can make this a lot shorter. Try this: 
```
const handleOnClick = () => {
    setShowModal(!showModal)
}
```

File: client/src/assets/close.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<svg id="cancel" xmlns="http://www.w3.org/2000/svg" width="30.411" height="30.411" viewBox="0 0 30.411 30.411">
+  <g id="Group_25" data-name="Group 25" transform="translate(0 0)">
```
[the remark this answers is not in the document]
Comment (SebastianStupp) [reply client/src/assets/close.svg:2]: Maybe I will edit the svg's on a later point :)

File: client/src/components/Modal.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Button from './Button';
+import Close from '../assets/close.svg';
+
+const CloseImage = styled.img`
+  align-self: flex-end;
+  margin-top: 8px;
+  margin-right: 8px;
```
[the remark this answers is not in the document]
Comment (SebastianStupp) [reply client/src/components/Modal.js:9]: yeah look s maybe a it cleaner 👍

File: client/src/components/Modal.js (hunk 1 above)
Comment (SebastianStupp) [reply client/src/components/Modal.js:56]: thx for the hint 👍

**Pull request merged**